.github/workflows/relay.yml

```diff
@@ -77,13 +79,15 @@ jobs:
         if: |
           github.ref == 'refs/heads/main'
             && github.event_name != 'pull_request'
+            && github.event_name != 'schedule'
         uses: docker/login-action@v1
         with:
           registry: ${{ env.GCR_REGISTRY_URL }}
           username: _json_key
           password: ${{ secrets.KEEP_TEST_GCR_JSON_KEY }}
 
       - name: Build and publish Docker Runtime Image
+        if: github.event_name != 'schedule'
```

Comment:
We could move the condition from here to the `push:` option. This way we would build the image and not push it (which would be consistent with the behaviour in other workflows).

Reply:
Sure, done in 5f76e08